### PR TITLE
Feature/testing

### PR DESCRIPTION
**Pull request by rzhang0720**
This PR fixes some of the test cases that are currently in RDFS.

With the exception of **ReplicationTest** and **ReadWriteTest**, all the tests should be working. This behavior can be verified by building RDFS and then running `/home/rdfs/build/test/RunAllTests`

### CODE REVIEW

File: test/integration/ReplicationTest.cc [hunk 1]
```diff
@@ -32,7 +32,7 @@ namespace {
         // system("head -c 5 temp > actual_testfile1234");
         // Check that its contents match.
         // TODO: This test will fail until we implement the file lengths meta-data tracking.
-        ASSERT_EQ(0, system("diff expected_testfile1234 actual_testfile1234 > /dev/null"));
+//        ASSERT_EQ(0, system("diff expected_testfile1234 actual_testfile1234 > /dev/null"));
```
Comment (LINKIWI) [inline test/integration/ReplicationTest.cc]: Is this deliberate?

**Review by LINKIWI — CHANGES_REQUESTED** [review]
Please unversion

`test/scripts/.gen_random.py.swp`
`test/scripts/out.txt`

**Comment by rzhang0720** [comment]
unversioned:
`test/scripts/.gen_random.py.swp`
`test/scripts/out.txt`

added check back in testReadWrite in ReplicationTest

File: test/integration/ReplicationTest.cc (hunk 1 above)
Comment (rzhang0720) [reply test/integration/ReplicationTest.cc]: good catch, I changed it back

**Review by LINKIWI — APPROVED** [review]

**Comment by LINKIWI** [comment]
approving to unblock - merge if CI is clean. I think this will merge cleanly

**Pull request merged**